src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToDefenceSpecCallbackHandler.java

```diff
@@ -348,7 +348,7 @@ private CallbackResponse aboutToSubmit(CallbackParams callbackParams, boolean v1
         if (V_2.equals(callbackParams.getVersion()) && caseData.isRejectDefendantPaymentPlanNo()) {
             response.state(CaseState.PROCEEDS_IN_HERITAGE_SYSTEM.name());
         } else {
-            if (v1 && featureToggleService.isSdoEnabled()) {
+            if (v1) {
```

Comment:
Are we able to move this code into the base callback handler and free up the V1 callback?